src/common/mem.c

```diff
@@ -331,6 +425,13 @@ umempobj_set_slab_desc(struct umem_pool *ph_p, struct umem_slab_desc *slab)
 		rc = dav_class_register((dav_obj_t *)ph_p->up_priv, &davslab);
 		/* update with the new slab id */
 		slab->class_id = davslab.class_id;
+		break;
+	case DAOS_MD_ADMEM:
+		/* NOOP for ADMEM now */
```

Comment:
You need to fake a slab ID for each registered slabs, it won't work otherwise.

Reply:
right, looks like the caller of umempobj_set_slab_desc() already set slab->class_id (see vos_register_slabs), let's see if will hit problem in following test.